skills_ml/job_postings/computed_properties/computers.py

```diff
@@ -89,35 +89,26 @@ class SOCClassifyProperty(JobPostingComputedProperty):
     """Classify the SOC code from a trained classifier
 
     Args:
-        s3_conn (boto.s3.connection)
         classifier_obj (object, optional) An object to use as a classifier.
             If not sent one will be downloaded from s3
     """
-    def __init__(self, s3_conn, classifier_obj=None, *args, **kwargs):
+    def __init__(self, classifier_obj=None, *args, **kwargs):
```

Comment:
It appears we are now depending on the classifer_obj instead of allowing to to be optional, so we should make the argument required.

Reply:
make sense.